### ISSUE TITLE
KeyboardInputGesture.send should use SendInput rather than Keybd_event

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4294]
**Reported by mdcurran on 2014-07-15 07:24**
Currently KeyboardInputGesture.send uses multiple Keybd_event calls to send a key press. It should be changed to use one singleSendInput call, which will guarantee that all key downs and key ups complete within the one call disallowing the possibility for the key downs to be sent, and then the user to press a key themselves, and then the key ups to be sent.

**Comment by jcsteh** [comment]
Status:

- This was implemented and then backed out due to it causing a regression; see https://github.com/nvaccess/nvda/issues/4294#issuecomment-155322168.
- The underlying cause of the regression might be less relevant after #7201.
- While this is probably the "right" thing to do, there's still no specific motivation for this at present, so the effort and risk involved is hard to justify.

Closing as won't fix for now. We can reopen if we find a specific reason to do this.
